### PR TITLE
Hit the Api

### PR DESCRIPTION
**Pull request by monicah79**
- Fetch data from API
- Implement functionality  in submit and refresh button 
- Use arrow function

### CODE REVIEW

File: src/modules/game.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      name: gameName,
+    }),
+  });
+  // .then((response) => {
+
+  // });
```
Comment (MahmoudBakr23) [inline src/modules/game.js]: **Optional**
- Please remove the unused code for a cleaner code base!

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+    },
+    body: JSON.stringify({
+      user: name,
+      score: scores,
+    }),
+  })
+    .then(() => {
+      refreshBtn.click();
+    });
+};
```
Comment (MahmoudBakr23) [inline src/index.js]: **Required**
- [ ] Please use `async await` with `fetch` when dealing (both send and receive) with the API as project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Use%20async%20and%20await%20JavaScript%20features%20to%20consume%20the%20API.) dictates. 😄

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @monicah79 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Nice work with the API so far
- Great file structure
- Descriptive PR
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Please fix the MIT link for a professional README file
_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — COMMENTED** [review]
### Hi @monicah79 😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

**Invalid Submission**

- Please make sure to push the changes requested by the previous reviewer before asking for another review. 🙏 

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._ 🙏

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

**Comment by monicah79** [comment]
hey i just push my lapttop has  problem with time thats why its showing long ago hours kindly check  on the up dates

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @monicah79 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
- [ ] Well done working on the requested changes and communicating. However, I'm afraid the problem isn't yet fixed! Kindly make the API call in the src/index.js file `async await` for making a `POST` request to the API!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

n/a

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @monicah79, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**